python/hsfs/feature_group.py

```diff
@@ -965,6 +970,29 @@ def get_validation_history(
                 "Only Feature Group registered with Hopsworks can fetch validation history."
             )
 
+    def enabled_statistics_monitoring(
+        self,
+        feature_name: str,
+        window_configuration_type: str,
+        time_offset: str,
+        window_length: str,
+        scheduler_config: str,
```

Comment:
Parameter `row_percentage` is missing in `enabled_statistics_monitoring.`

Also, `time_offset`, `window_length` and `row_percentage` should be nullable I believe.